src/reactivity/unwrap.ts

```diff
@@ -29,8 +29,8 @@ export function unwrapRefProxy(value: any, map = new WeakMap()) {
 
   for (const k of Object.keys(value)) {
     const r = value[k]
-    // don't unwrap or do anything to raw
-    if (isRaw(r)) {
+    // don't process on falsy or raw
+    if (!r || isRaw(r)) {
```

Comment:
Would it better to check falsy inside isRaw?

Reply:
```js
export function isRaw(obj: any): boolean {
  return !!obj && hasOwn(obj, RawIdentifierKey) && obj[RawIdentifierKey] === RawIdentifier;
}
```
Not sure if is will help, the idea is if is `isRaw` it shouldn't do anything to the object.

even if I add that condition on `isRaw` I still need to do `unwrap`, since we don't need to process `falsy` values